src/subpages/account.tsx

```diff
@@ -151,112 +155,67 @@ const AccountPage: React.FC<AccountPageProps> = ({ location }) => {
-        <Link href={`../transaction/?${id}`}>{id}</Link>
-      ),
-    },
-    {
-      key: 'columnSignature',
-      name: 'Signature',
-      fieldName: 'signature',
-      minWidth: 100,
-      maxWidth: 200,
-      isRowHeader: true,
-      isResizable: true,
-      isSorted: false,
-      isSortedDescending: true,
-      data: 'number',
-      isPadded: true,
-    },
-    {
-      key: 'columnSigner',
-      name: 'Signer',
-      fieldName: 'signer',
-      minWidth: 100,
-      maxWidth: 200,
-      isRowHeader: true,
-      isResizable: true,
-      isSorted: false,
-      isSortedDescending: true,
-      data: 'number',
-      isPadded: true,
-      onRender: ({ signer }: Transaction) => (
-        // FIXME: We'd better to use absolute paths and make Gatsby automatically
-        // to rebase these absolute paths on the PATH_PREFIX configuration.
-        <Link href={`./?${signer}`}>{signer}</Link>
-      ),
-    },
-    {
-      key: 'columnTimestamp',
-      name: 'Timestamp',
-      fieldName: 'timestamp',
-      minWidth: 100,
-      maxWidth: 200,
-      isRowHeader: true,
-      isResizable: true,
-      isSorted: false,
-      isSortedDescending: true,
-      data: 'number',
-      isPadded: true,
-      onRender: ({ timestamp }: Transaction) => (
-        <Timestamp timestamp={timestamp} />
-      ),
-    },
-  ];
+interface TransactionListWrapProps {
+  signed?: Transaction[];
+  involved?: Transaction[];
+  missingNonces?: number[];
+  loading: boolean;
+}
 
-  return (
-    <DetailsList
-      items={transactions}
-      columns={columns}
-      selectionMode={SelectionMode.none}
-      getKey={(tx: Transaction) => tx.id}
-      setKey="set"
-      layoutMode={DetailsListLayoutMode.justified}
-      isHeaderVisible={true}
-      // FIXME: We'd better to use absolute paths and make Gatsby automatically
-      // to rebase these absolute paths on the PATH_PREFIX configuration.
-      onItemInvoked={({ id }: Transaction) => navigate(`../transaction/?${id}`)}
+const TransactionListWrap: React.FC<TransactionListWrapProps> = ({
+  signed,
+  involved,
+  missingNonces,
+  loading,
+}) => (
+  <>
+    <h2>Signed Transactions {counter(signed)}</h2>
+    <TransactionList
+      loading={loading}
+      transactions={signed ? signed : null}
+      notFoundMessage={'No Signed Transactions'}
     />
-  );
-};
+    <h2>Involved Transactions {counter(involved)}</h2>
+    <TransactionList
+      loading={loading}
+      transactions={involved ? involved : null}
+      notFoundMessage={'No Involved Transactions'}
+    />
+    <h2>Missing Nonces {counter(missingNonces)}</h2>
+    {missingNonces ? (
+      missingNonces.length > 0 ? (
+        missingNonces.map(nonce => <p>{nonce}</p>)
+      ) : (
+        <div>No missing nonces.</div>
+      )
+    ) : (
+      'Loading..'
+    )}
+  </>
+);
 
-export default AccountPage;
+function counter(items?: any[]) {
+  return items !== undefined && items.length > 0 && `: ${items.length}`;
```

Comment:
Is it still useful to count the number in pagenation view?

Reply:
I believe the counter is still useful and it should show the total number of items in *every* page, not only items in this single page.

Reply:
@dahlia  Would it be better to count all mined blocks?